server.R

```diff
@@ -689,7 +689,7 @@ server <- function(input, output, session) {
         )
         updateSelectInput(session,
```

Comment:
Don't think this updateSelectInput is actually doing anything. For a start "inSelect" is a selectizeInput rather than a selectInput and also levels_v doesn't look like it gets updated based on any of the inputs anywhere, so I don't think there's never any changes to update the selectizeInput choices with anyway.

It's not breaking anything though, so don't think it's massively important to change now if you want to avoid editing the code as much as possible just prior to going live.